portfolio/src/main/webapp/style.css

```diff
@@ -98,6 +98,7 @@ label[for="max-display"] {
     font-family: "Lato", 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
     color: #2E4A62;
     background-color: #F0EDE5;
+    margin: 0 0 10px 10px;
```

Comment:
For the bottom margin here, I don't think you should rely on the bottom margin of one element to be responsible for the vertical spacing for the entire row. It would be better to style the container of all the elements in the row.

Reply:
I specified line heights for the comment section to ensure the spacing. This margin here is actually for the delete button on the bottom of the comment section so that there's space between it and the left bottom corner. But I'll give the delete button an ID to specify that:))